start.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+RUN mix deps.get && \
+    cd assets && \
+    npm install && \
```

Comment:
Is it the Docker's command @ankitkalia1195 is it build to the Docker image 🤔  or it runs every time we start the app 🤔

Reply:
I will move it to Dockerfile. My original intention was that if we change some deps in codebase, we will need to run these commands again or build the docker image again. But maybe dev can run them manually inside the container.